resources/js/Pages/Results.vue

```diff
@@ -1,28 +1,51 @@
 <template>
     <div class="page-container results-page">
         <Head title="Mismatch Finder - Results" />
-        <section id="results" v-if="Object.keys(results).length">
-            <section class="item-mismatches" v-for="(mismatches, item, idx) in results" :key="idx">
+        <div v-if="Object.keys(results).length < item_ids.length" class="not-found">
+            <div id="top-button">
+                <wikit-button
+                    variant="normal"
+                    type="neutral"
+                    native-type="button"
+                    @click.native="returnHome"
+                >
+                    &#x2190; {{ $i18n('refine-items-selection') }}
+                </wikit-button>
+            </div>
+            <section id="description-section">
+                <h2 class="h4">{{ $i18n('results-page-title') }}</h2>
+                <p id="about-description" >{{ $i18n('results-page-description') }}</p>
+            </section>
+            <section id="message-section">
+                <Message type="notice">
+                    <span>{{ $i18n('no-mismatches-found-message') }} 
+                        <span class="message-link" v-for="(label, item_id) in labelsFromNotFound" :key="item_id">
+                            <wikit-link 
+                                :href="`http://www.wikidata.org/wiki/${item_id}`">{{label}} ({{item_id}})
+                            </wikit-link>
+                        </span>
```

Comment:
Instead of going through the entire labels object just to find the labels for a certain Id, we can just directly access it here, like so:

```suggestion
                        <span class="message-link" v-for="item_id in labelsFromNotFound" :key="item_id">
                            <wikit-link 
                                :href="`http://www.wikidata.org/wiki/${item_id}`">{{labels[item_id]}} ({{item_id}})
                            </wikit-link>
                        </span>
```

If you accept this change, we can also change the name of the function that returns the not found item ids.